src/EPR.Calculator.API/Program.cs

```diff
@@ -42,14 +44,28 @@
 builder.Services.AddScoped<ICalcResultsExporter<CalcResult>, CalcResultsExporter>();
 builder.Services.AddScoped<ICalcResultLapcapDataBuilder, CalcResultLapcapDataBuilder>();
 builder.Services.AddScoped<ICalcResultSummaryBuilder, CalcResultSummaryBuilder>();
-builder.Services.AddScoped<IStorageService, BlobStorageService>();
```

Comment:
Was switching from blob to local storage intended?  Afaik it's only supposed to be used for local testing.

Reply:
Good point, It was an error. Thanks @tim-powell-eviden